metabase/metabase
# fix static-viz type errors

### Description

Fixes the remaining type errors in the `static-viz` directory, and also removes some unused code (`IsomorphicStaticChart.tsx`)

### How to verify

CI - confirm loki specs pass

`yarn type-check` confirm there are no more type errors in `static-viz` directory
